monero-outreach-docs/translations/es/merchants/merchant_faq_es.md

```diff
@@ -0,0 +1,58 @@
+
+### _¿Qué diferencia hay entre aceptar Monero a aceptar Bitcoin?_
+
+La configuración tanto de Bitcoin como de Monero son similares, pero las cadenas de bloques son diferentes. Monero tiene algunas características únicas que le podrían interesar más a los comerciantes que son:
+
+- **Pagos más rápidos** - Monero tiene un tiempo de verificación de 2 minutos por bloque a diferencia de los 10 minutos por bloque en Bitcoin. En promedio, tu transacción llegaría 5 veces más rápido a la cadena de bloques de Monero que a la de Bitcoin.
+- **Menos costoso** - Monero tiene comisiones mucho más bajas que Bitcoin y además, Monero continúa siendo cada vez más eficiente, la tendencia en términos de comisiones es tener comisiones más bajas que las de Bitcoin.
+- **No hay retrasos en las transacciones** - Los bloques de Monero son dinámicos mientras que Bitcoin tiene bloques fijos. Cuando la red de Bitcoin está ocupada y los bloques están llenos, las transacciones deben, o pagar comisiones altas, o retrasarse.
+- **Libre de riesgo** - Siendo que Monero es privado por defecto y por ende fungible, nadie puede discriminar el historial de las monedas. Las monedas marcadas como contaminadas en la red de Bitcoin es un problema creciente para los comerciantes, pero Monero no tiene ese problema.
+
+### _¿Cómo se comparan las comisiones de Monero con las comisiones de las tarjetas de crédito?_
+
+Las comisiones en la red de Monero son pagadas por quien envía la transacción. Recibir XMR siempre será gratis. Al cliente le cuesta aproximadamente $0.02 centavos de dólar pagar con Monero. Pagar con Monero es menos costoso que una transacción con tarjeta de crédito, además de tener un procesamiento mucho más rápido.
+
+| Comisión | American Express | Mastercard | Visa | Débito |
+|--|--|--|--|--|
+| Comisión de Procesamiento | 2.5% - 3.5% | 1.55% - 2.6% | 1.43% - 2.4% | 0.30% - 2.75% |
+| Comisión de intercambio | -- | 0.08% | 0.09% | 0.05% + $0.21 0.70% + $0.15 |
+| Comisión de la red | 0.15% | 0.08% | 0.09% | 0.75% - 0.90% |
+| Comisión Switch | -- | -- | -- | $0.025 - $0.075 |
+| Comisiones por compras de $10 USD | **$0.27 - $0.37** | **$0.18 - $0.28** | **$0.16 - $0.26** | **$0.35 - $0.66** |
+| Comisiones por compras de $100 USD | **$2.65 - $6.65** | **$1.78 - $2.83** | **$1.61 - $2.58** | **$3.45 - $6.60** |
+| Comisión por compras de $1000 USD | **$26.50 - $66.50** | **$17.80 - $28.30** | **$16.10 - $25.80** | **$34.50 - $66.00** |
+
+Las comisiones de las transacciones débito o crédito de los comerciantes varía según el tipo de negocio y de quien procesa el pago, pero una transacción de Monero será entre 7 veces y/o 3.023 veces menos costosa que la de una transacción con tarjeta débito o crédito.
```

Comment:
I'd change this:  "transacciones débito o crédito"  to this: "transacciones de débito o crédito"

and I'd say: 3023 instead of 3.023

Reply:
Thanks :). Done.